src/containers/widget/index.tsx

```diff
@@ -77,8 +77,15 @@ const WidgetWrapper: React.FC<WidgetLayoutProps> = (props: WidgetLayoutProps): n
             </h2>
             <WidgetControls id={id} />
           </header>
-
-          {!widgetsCollapsed[id] && children}
+          <div
+            className={cn({
+              hidden: widgetsCollapsed[id],
+              'display-block': !widgetsCollapsed[id],
```

Comment:
I don't think the class `display-block` exists in Tailwind, shouldn't be `block`?